app/controllers/application_controller.rb

```diff
@@ -49,9 +49,12 @@ def check_form
   end
 
   def check_draw_period
+    url = request.fullpath
     unless current_draw_period != nil then
       unless current_user.is_admin then
-        redirect_to coming_soon_path
+        unless url == "/" or url.start_with?("/dorm") #Let students see the map but nothing else
```

Comment:
Should students also be able to see the students list?  They can see it during normal room draw, and I don't see a reason to revoke their access after.

Reply:
I don't see a problem with them looking at the student list.